docs/source/howto/run_codes.rst

```diff
@@ -399,35 +399,37 @@ See :ref:`topics:processes:usage:launching` and :ref:`topics:processes:usage:mon
 
 .. _how-to:run-codes:caching:
 
-How to save computational resources using caching
-=================================================
+How to save compute time with caching
+=====================================
 
-There are numerous reasons why you might need to re-run calculations you have already run before.
-
-    If you run a great number of complex workflows in high-throughput, that each may repeat the same calculation, or you have to restart an entire workflow that failed somewhere half-way through.
+In practice, one often ends up re-running calculations one has already run before - be it because two workflows include the same calculation or because one needs to restart a workflow that failed due to some infrastructure problem.
```

Comment:
```suggestion
Over the course of a project, you may end up re-running the same calculations multiple times - be it because two workflows include the same calculation or because one needs to restart a workflow that failed due to some infrastructure problem.
```